cmd/rosetta/cli.go

```diff
@@ -0,0 +1,176 @@
+package main
+
+import (
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/urfave/cli"
+)
+
+var (
+	helpTemplate = `NAME:
+   {{.Name}} - {{.Usage}}
+USAGE:
+   {{.HelpName}} {{if .VisibleFlags}}[global options]{{end}}
+   {{if len .Authors}}
+AUTHOR:
+   {{range .Authors}}{{ . }}{{end}}
+   {{end}}{{if .Commands}}
+GLOBAL OPTIONS:
+   {{range .VisibleFlags}}{{.}}
+   {{end}}
+VERSION:
+   {{.Version}}
+   {{end}}
+`
+
+	cliFlagPort = cli.IntFlag{
+		Name:  "port",
+		Usage: "Specifies the TCP port used by Rosetta endpoints.",
+		Value: 9090,
+	}
+
+	cliFlagOffline = cli.BoolFlag{
+		Name:  "offline",
+		Usage: "Starts in offline mode",
+	}
+
+	cliFlagLogLevel = cli.StringFlag{
+		Name: "log-level",
+		Usage: "Specifies the logger `level(s)`. It can contain multiple comma-separated value. For example" +
+			", if set to *:INFO the logs for all packages will have the INFO level. However, if set to *:INFO,api:DEBUG" +
+			" the logs for all packages will have the INFO level, excepting the api package which will receive a DEBUG" +
+			" log level.",
+		Value: "*:" + logger.LogInfo.String(),
+	}
+
+	cliFlagLogsFolder = cli.StringFlag{
+		Name:  "logs-folder",
+		Usage: "Specifies where to save the log files.",
+		Value: "",
+	}
+
+	cliFlagObserveActualShard = cli.UintFlag{
+		Name:  "observe-actual-shard",
+		Usage: "Specifies the actual shard to observe.",
+		Value: 0,
+	}
+
+	cliFlagObserveProjectedShard = cli.UintFlag{
+		Name:  "observe-projected-shard",
+		Usage: "Specifies the projected shard to observe.",
+		Value: 0,
+	}
+
+	cliFlagObserver = cli.StringFlag{
+		Name:  "observer",
+		Usage: "Specifies the URL of the observer.",
+		Value: "http://localhost:10100",
+	}
+
+	cliFlagObserverPubKey = cli.StringFlag{
+		Name:  "observer-pubkey",
+		Usage: "Specifies the public key of the observer.",
+		Value: "0000000000000000000000000000000000000000000000000000000000000000",
+	}
+
+	cliFlagChainID = cli.StringFlag{
+		Name:  "chain-id",
+		Usage: "Specifies the Chain ID.",
+		Value: "local-testnet",
+	}
+
+	cliFlagNumShards = cli.UintFlag{
+		Name:  "num-shards",
```

Comment:
all the configs (num shards, chain ID, min gas price, min gas limit, gas per data byte) can be fetched `observer/network/config`. 
regarding the genesis block, it's quite a good idea actually to include it as well in that endpoint